src/logic/basic.lean

```diff
@@ -816,6 +816,10 @@ exists_congr (λ a, exists₃_congr (h a))
 theorem forall_swap {p : α → β → Prop} : (∀ x y, p x y) ↔ ∀ y x, p x y :=
 ⟨swap, swap⟩
 
+/-- We intentionally restrict the type of `α` here so that this is a safer to use in simp. -/
```

Comment:
```suggestion
/-- We intentionally restrict the type of `α` here so that this is safer to use in simp. -/
```
Also, I am not sure this should be a docstring, as the comment refers to the previous lemma but is not standalone. Convert `/--` to `/-`?